core/domain/classifier_domain.py

```diff
@@ -277,6 +277,27 @@ def status(self):
     def training_data(self):
         return self._training_data
 
+    def update_state_name(self, state_name):
```

Comment:
@anmolshkl . I'm not sure I understand. We did plan to update status inside the controller functions right? I don't see how this could be a problem.